### ISSUE TITLE
Is it possible to render fisheye or panorama image?

### ISSUE THREAD

**Issue by StarRealMan** [owl-project/NVISII#166]
Thank you for your great work!
I just want to know if it is possible to render fisheye or panorama image by setting the camera.
I tried to set the fov of the camera greater than 180 and this is the image I get:
![02_random_scene](https://github.com/owl-project/NVISII/assets/32333352/eed77637-5651-490c-bb41-7a3c96f25952)

aspect ratio: 1
fov: 220

It dose not seem to be OK.

Thank you if you can help!

**Comment by StarRealMan** [comment]
I found that the `fov` is set using `glm::perspective()` which uses a pinhole camera model, so, I think it is not possible to achieve fisheye effect in this way.
I stil wonder if anyone can come up with ideas about fisheye image rendering using NVISII. Thanky you if you can reply.

**Comment by TontonTremblay** [comment]
I remember @natevm and I talking about, and I recall that he said he would not be too hard to add, but at the moment I was not sure we would ever need it, the prime application we had in mind was computer vision to which you normally get rectified images and thus you would never get rounding artifacts. 

I labeled it as a feature request.

**Comment by natevm** [comment]
Yeah, in theory it would be possible to do. I vaguely remember a ray tracing gem on how to do a fish eye lens, but I’m not sure what changes would need to be made to the camera component to make that work. 

I would also like to have an orthographic camera…

**Comment by StarRealMan** [comment]
Thank you for your timely reply!
I'm now trying to generate fisheye images using cubemap rendered from NVISII and hope this will work. The only problem I need to worry is the mismatch of the pixels. I think greater resolution may solve it.
